README.md

```diff
@@ -1,4 +1,4 @@
-# Opdracht beschrijving
+w# Opdracht beschrijving
```

Comment:
deze 'w' is verdwaald :(

Reply:
neat. misschien kun je andere alternatieven voor if-else onderzoeken (zoals met bijvoorbeeld). maar het is prima in het algemeen. cheers